torch/_lazy/__init__.py

```diff
@@ -0,0 +1,22 @@
+import torch._C._lazy
+
+
+def mark_step(device: str = "lazy:0", wait=False):
```

Comment:
is the plan to expose all LTC python functions via `torch._lazy`? doesn't the underscore usually imply "private" and user shouldn't call? or is it eventually going to something like `torch.lazy` and the API exposed more similarly to how its currently done on the staging branch?

Reply:
yea, torch._lazy signifies it is not a stable API, that's our convention.  It can't be torch.lazy until we (as a team) make a stronger commitment to long term support, etc.